multires-lyrics-search/tests/test_multireslyrics.py

```diff
@@ -58,9 +60,35 @@ def get_flow():
 
 @pytest.fixture
 def queries():
-    return [("Trudging slowly\n", ["Don't say anything\n", "Check that: I'll watch that game.\n", "I'm trying to talk\n"]),
-            ('I could feel at the time\n', ['Sorry\n', 'Hey\n', 'Allб\n']),
-            ("I promise.\n", ["Check that: I'll watch that game.\n", "I just think it's a fabulous app\n", 'Could you get that\n'])]
+    return [
```

Comment:
what is the `structure` of these `queries`? Aren't queries supposed to be provided as simple `strings`?

Reply:
ah I see, can we rename `queries_with_results`?